todo_rxbloc_app/lib/feature_manage_todo/bloc/manage_todo_bloc_extensions.dart

```diff
@@ -0,0 +1,40 @@
+part of 'manage_todo_bloc.dart';
+
+extension _ManageTodoExtension on Stream {
+  Stream<Result<NewTodoEnum>> createTodo(
+    Stream<String> setTitle,
+    Stream<String> setDescription,
+    ManageTodoService manageService,
+  ) =>
+      withLatestFrom2<String, String, TextFieldsData>(
```

Comment:
Please try to use the following snippet directly in the bloc. I'm not sure whether this `createTodo` custom operator brings a lot of value

 `manageService.createTodo(value.title, value.description)`